python/scripts/archive_repositories.py

```diff
@@ -31,84 +44,69 @@ def get_commit(repository):
 
 
 def ready_for_archiving(repository, archive_date) -> bool:
-    """See if repository is ready for archiving based on last commit date
-
-    Args:
-        repository (Repository): the repository object
-        archive_date (datetime): the archive date
-
-    Returns:
-        bool: true if the last commit date is longer than the archive date
-    """
     commit = get_commit(repository)
     if commit == 0:
-        print("Manually check repository: " + repository.name)
-    else:
-        if commit.commit.author.date < archive_date:
-            time.sleep(1)
-            return True
+        logging.warning(f"Manually check repository: {repository.name}")
+    elif commit.commit.author.date < archive_date:
+        return True
     return False
 
 
-def main():
-    # Logging Config
-    logging.basicConfig(
-        format="%(asctime)s %(levelname)-8s %(message)s",
-        level=logging.INFO,
-        datefmt="%Y-%m-%d %H:%M:%S",
-    )
-
-    # TODO: Change this to point at a different GitHub organization
-    organization = "ministryofjustice"
-    org_token = os.getenv("ADMIN_GITHUB_TOKEN")
-    if not org_token:
+def get_environment_variables() -> tuple[str, str]:
+    github_token = os.getenv("ADMIN_GITHUB_TOKEN")
+    if not github_token:
         raise ValueError(
             "The env variable ADMIN_GITHUB_TOKEN is empty or missing")
 
-    # How long ago in which the repositories should be archived
-    archive_date_days = 0
-    archive_date_months = 6
-    archive_date_years = 1
+    github_organization_name = os.getenv("GITHUB_ORGANIZATION_NAME")
+    if not github_organization_name:
+        raise ValueError(
+            "The env variable GITHUB_ORGANIZATION is empty or missing")
+
+    return github_token, github_organization_name
+
 
-    archive_date = datetime.now() - relativedelta(
-        days=archive_date_days, months=archive_date_months, years=archive_date_years
-    )
+def get_config_for_organization(github_organization_name: str) -> tuple[str, list[str], str] | ValueError:
```

Comment:
I will do some more refactoring inside this function and the components being used.

The purpose of this PR was just to get the script in a state to handle multiple orgs and configs 🤹